src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.HttpContentAsyncStream.cs

```diff
@@ -142,6 +142,15 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
                 }
             }
 
+            public override void Write(byte[] buffer, int offset, int count)
+            {
+                // WriteAsync should be how writes are performed on this stream as
+                // part of HttpContent.CopyToAsync.  However, we implement Write just
```

Comment:
Actually, derived classes of the abstract `HttpContent` class do not implement `CopyToAsync`.  That is a public method which actually calls protected methods such as `SerializeToStreamAsync`. That is the method that WCF and others override. So, I suggest that the comments be revised.